### PR TITLE
[zh] sync docs/reference/labels-annotations-taints/_index.md

### PR DESCRIPTION
**Pull request by bzsuni**
sync docs/reference/labels-annotations-taints/_index.md

### CODE REVIEW

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -185,6 +185,136 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
...
+Used on: Objects being used as ApplySet parents.
+
+Use of this annotation is alpha.
+For Kubernetes version {{< skew currentVersion >}}, you can use this annotation on Secrets, ConfigMaps, or custom resources if the {{< glossary_tooltip term_id="CustomResourceDefinition" text="CustomResourceDefinition" >}} defining them has the `applyset.kubernetes.io/is-parent-type` label.
+
+Part of the specification used to implement [ApplySet-based pruning in kubectl](/docs/tasks/manage-kubernetes-objects/declarative-config/#alternative-kubectl-apply-f-directory-prune). This annotation is applied to the parent object used to track an ApplySet to extend the scope of the ApplySet beyond the parent object's own namespace (if any). The value is a comma-separated list of the names of namespaces other than the parent's namespace in which objects are found.
+-->
+### applyset.kubernetes.io/additional-namespaces (alpha) {#applyset-kubernetes-io-additional-namespaces}
+
+示例： `applyset.kubernetes.io/additional-namespaces: "namespace1,namespace2"`
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:202]:
```suggestion
示例：`applyset.kubernetes.io/additional-namespaces: "namespace1,namespace2"`
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -185,6 +185,136 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
...
+
+用于： 作为 ApplySet 父对象使用的对象。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:204]:
```suggestion
用于：作为 ApplySet 父对象使用的对象。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -185,6 +185,136 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
...
+
+此注解的使用处于 alpha 版本。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:206]:
```suggestion
此注解处于 Alpha 阶段。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 4]
```diff
@@ -185,6 +185,136 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
...
+对于 Kubernetes {{< skew currentVersion >}} 版本， 如果定义它们的 {{< glossary_tooltip term_id="CustomResourceDefinition" text="CustomResourceDefinition" >}} 有 `applyset.kubernetes.io/is-parent-type` 标签， 那么您可以在 Secret、 ConfigMaps 或自定义资源上使用此注解。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:207]:
```suggestion
对于 Kubernetes {{< skew currentVersion >}} 版本，如果定义它们的
{{< glossary_tooltip term_id="CustomResourceDefinition" text="CustomResourceDefinition" >}}
打了 `applyset.kubernetes.io/is-parent-type` 标签，那么你可以在 Secret、ConfigMap 或自定义资源上使用此注解。
```
请给长句适当折行，方便 review

**Review by windsonsea — COMMENTED** [review]
请阅读[中文本地化风格指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#layout-format)

**Comment by bzsuni** [comment]
Thank you very much. I have made modifications based on your suggestion @windsonsea

**Comment by windsonsea** [comment]
Note this page is refactoring in #41540.

**Comment by windsonsea** [comment]
#41540 is merged. Could you please include those changes too? @bzsuni

**Comment by tengqm** [comment]
Looks like this has no conflict with the 'main' branch?

**Comment by windsonsea** [comment]
Yes, no conflict. This file is [frequently changed recently with large changes](https://github.com/kubernetes/website/commits/main/content/en/docs/reference/labels-annotations-taints/_index.md) more than 1000 lines and similar zh PR #41902 is opened too.

Merging this can move a step forward, IMO.

**Pull request merged**